arangod/Metrics/MetricsFeature.cpp

```diff
@@ -155,19 +232,32 @@ void MetricsFeature::toPrometheus(std::string& result, CollectMode mode) const {
       batch->toPrometheus(result, _globals, _ensureWhitespace);
     }
   }
-  auto& sf = server().getFeature<StatisticsFeature>();
-  auto time = std::chrono::duration<double, std::milli>(
-      std::chrono::system_clock::now().time_since_epoch());
-  sf.toPrometheus(result, time.count(), _globals, _ensureWhitespace);
-  auto& es = server().getFeature<EngineSelectorFeature>().engine();
-  if (es.typeName() == RocksDBEngine::kEngineName) {
-    es.toPrometheus(result, _globals, _ensureWhitespace);
+
+  if (metricsParts.includeStandardMetrics()) {
```

Comment:
This looks strange, since the same `if` statement is done 4 times in a row. Do we only need one `if` here?